esphomeRemote/esphomeRemoteLight.h

```diff
@@ -2,30 +2,172 @@
 #include "DisplayUpdateInterface.h"
 #include "MenuTitle.h"
 #include "FriendlyNameEntity.h"
+#include <sstream>

```

Comment:
just curious why do we need sstream

Reply:
we don't. I used it for debugging and forgot to remove it.

Reply:
can you remove this so we can close this up

Reply:
done